pkgs/development/python-modules/devpi-ldap/default.nix

```
@@ -0,0 +1,61 @@
+{
+  lib,
+  buildPythonPackage,
+  devpi-server,
+  fetchFromGitHub,
+  ldap3,
+  mock,
+  pytest-cov,
+  pytest-flake8,
+  pytestCheckHook,
+  python3,
+  pythonOlder,
+  pyyaml,
+  setuptools,
+  webtest,
+}:
+
+buildPythonPackage rec {
+  pname = "devpi-ldap";
+  version = "2.1.1-unstable-2023-11-28";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "devpi";
+    repo = "devpi-ldap";
+    rev = "281a21d4e8d11bfec7dca2cf23fa39660a6d5796";
+    hash = "sha256-vwX0bOb2byN3M6iBk0tZJy8H39fjwBYvA0Nxi7OTzFQ=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    devpi-server
+    pyyaml
+    ldap3
+  ];
+
+  nativeCheckInputs = [
+    devpi-server
+    mock
+    pytest-cov
```

```suggestion
    pytest-cov-stub
```

Let's use the stub, we don't actually need coverage testing.